docstore/drivertest/drivertest.go

```diff
@@ -1254,17 +1275,49 @@ func testGetQueryKeyField(t *testing.T, coll *ds.Collection, revField string) {
 
 func sortByKeyField(d1, d2 docmap) bool { return d1[KeyField].(string) < d2[KeyField].(string) }
 
+// TODO(shantuo): consider add this test to all action tests, like the AltRev
```

Comment:
I didn't integrate this test into all other action tests since there are a few PR's touching the conformance test currently and this can wait I think.

Reply:
OK.